firmware/src/system_config/mx_795_512l/system_config.h

```diff
@@ -102,6 +102,11 @@ SUBSTITUTE GOODS, TECHNOLOGY, SERVICES, OR ANY CLAIMS BY THIRD PARTIES
 
 
 
+

```

Comment:
This is an autogenerated file. The mystery of why is known only to the harmony developers themselves!